packages/hebao_v1/contracts/modules/core/ForwarderModule.sol

```diff
@@ -74,8 +74,7 @@ abstract contract ForwarderModule is BaseModule
 
             // We only allow the wallet to call itself to addModule
             (to == from) &&
-            data.toBytes4(0) == Wallet.addModule.selector &&
-            controllerCache.walletRegistry.isWalletRegistered(from) ||
+            data.toBytes4(0) == Wallet.addModule.selector ||
```

Comment:
Unnecessary check I think. If `from` isn't a wallet than that should be okay, `addModule` will be called on this arbitrary (but similar to a Wallet) contract which is fine.